app/decorators/course_decorator.rb

```diff
@@ -184,6 +184,10 @@ def cycle_range
     "#{course.recruitment_cycle.year} to #{course.recruitment_cycle.year.to_i + 1}"
   end
 
+  def academic_year
```

Comment:
Can we move this down to recruitment cycle and just delegate down

Reply:
Sorry, not sure what you mean?

Reply:
oh the code has changed...

can you move it done to `Course` model

so we can have `course.academic_year` so that this decorator can just utilise `delegate_all`